components/pages/basicPageLayout/BasicPageLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+import ParagraphResolver from "@/components/paragraphs/ParagraphResolver"
+import { NodeGoPage } from "@/lib/graphql/generated/dpl-cms/graphql"
+
+function BasicPageLayout({ pageData }: { pageData: NodeGoPage }) {
+  const { paragraphs } = pageData
+
+  return (
+    <div className="flex flex-col gap-y-paragraph-spacing">
+      <ParagraphResolver paragraphs={paragraphs ?? []} />
+    </div>
+  )
```

Comment:
I think that if we beforehand know that there is no data, in this case paragraphs I think we should not to to resolve pragraphs.
So it would be:
```typescript
  if (!paragraphs) {
    // Null or something else default.
    return nulll;
 }
  return (
    <div className="flex flex-col gap-y-paragraph-spacing">
      <ParagraphResolver paragraphs={paragraphs} />
    </div>
  )
```

Reply:
What kind of rendering do we want if there are no paragraphs?